From 1853994c00ca6ff112abd7a23504d6736995081a Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Wed, 27 Sep 2023 09:21:38 +0200
Subject: [PATCH] Update app-shell.role.json.in: Add trustLevel

Fixes:

Sep 27 07:51:47 qemux86-64 ls-hubd[290]: [] [pmlog] ls-hubd LSHUB_ROLE_FILE {"FILE":"file_parser.cpp","LINE":195} No trust level specified for application in role file (/usr/share/luna-service2/roles.d/app-shell.role.json)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 src/webos/install/app_shell/app-shell.role.json.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/webos/install/app_shell/app-shell.role.json.in b/src/webos/install/app_shell/app-shell.role.json.in
index 35ae30c16b..b0bc125a2e 100644
--- a/src/webos/install/app_shell/app-shell.role.json.in
+++ b/src/webos/install/app_shell/app-shell.role.json.in
@@ -1,5 +1,6 @@
 {
     "exeName": "/usr/bin/app-shell/app_shell",
+    "trustLevel" : "oem",
     "type": "privileged",
     "allowedNames": [
         "com.webos.app.enactbrowser",
